targets/TARGET_Samsung/security_subsystem/api/trng_api.c

```diff
@@ -0,0 +1,53 @@
+/****************************************************************************
+ *
+ * Copyright 2019 Samsung Electronics All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+#include "hal/trng_api.h"
+#include "sss_driver_rng.h"
```

Comment:
@ARMmbed/mbed-os-crypto Please review

Reply:
Hi, @ARMmbed/mbed-os-crypto team
Could you please review this PR from your team's point of view? FYI. This code has passed all the MbedTLS Greentea test and netsocket TLS tests. Reference: https://github.com/ARMmbed/Samsung_projects/issues/112#issuecomment-570130219